<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>heroPage</class>
 <widget class="QWidget" name="heroPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>727</width>
    <height>548</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>711</width>
     <height>511</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QListWidget" name="starterItems">
        <item>
         <property name="text">
          <string>Game Start:</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="earlyGame">
        <item>
         <property name="text">
          <string>Early Game:</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QListWidget" name="midGame">
        <item>
         <property name="text">
          <string>Mid Game:</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="lateGame">
        <item>
         <property name="text">
          <string>Late Game:</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
